### PR TITLE
Autify upload 0.0.1

### PR DESCRIPTION
**Pull request by tarappo**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/autify-upload/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+support_url: https://github.com/autifyhq/bitrise-steps-autify-upload/issues
+published_at: 2021-06-02T05:10:23.914108+09:00
+source:
+  git: https://github.com/autifyhq/bitrise-step-autify-upload.git
+  commit: 5d9bc7aba74c96d11c5cd4098b82e1537350f5c9
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- android
```
Comment (koral--) [inline steps/autify-upload/0.0.1/step.yml:20]: It contradicts with a description.

Comment (tarappo) [reply steps/autify-upload/0.0.1/step.yml]: thanks.
I removed this tag.

Comment (sotayamashita) [reply steps/autify-upload/0.0.1/step.yml]: @koral-- Could you check it out again. Thanks in advance

File: steps/autify-upload/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
```
Comment (lpusok) [inline steps/autify-upload/0.0.1/step.yml:30]: is_requires_admin_user is deprecated and can be removed.

File: steps/autify-upload/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  This Step, however, does NOT generate your build: to create a `.app` file, you need the right step. For example, `Xcode build for simulator` Step or any other Step can generate a build. Add task URL
+website: https://github.com/autifyhq/bitrise-steps-autify-upload
+source_code_url: https://github.com/autifyhq/bitrise-steps-autify-upload
+support_url: https://github.com/autifyhq/bitrise-steps-autify-upload/issues
+published_at: 2021-06-02T19:27:14.276215+09:00
+source:
+  git: https://github.com/autifyhq/bitrise-step-autify-upload.git
+  commit: 4e2d9cdd6a278cbb08aaf9b8276438ed6b6e9396
+host_os_tags:
+- osx-10.10
```
Comment (lpusok) [inline steps/autify-upload/0.0.1/step.yml:17]: `host_os_tags` is deprecated and no longer checked. I recommend to check that the Step also works on Linux (even if not likely to be used) and removing it, in a future release.

**Review by lpusok — CHANGES_REQUESTED** [review]
Thanks for the contribution!
The source code URL needs to be updated before merging. For this I recommend opening an other PR.
Also left some recommendations.
Noticed that there are test files in the Step repository, this is not a blocking problem. But keep in mind that the included artifacts will also be cloned/copied in every Step run, so it is a good idea to consider moving them in a separate repository.

File: steps/autify-upload/0.0.1/step.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  Autify Upload
+summary: |
+  Uploads your build to Autify.
+description: |-
+  Uploads you build `.app` to [Autify](https://autify.com/mobile) using Autify's API. You can upload the iOS build with the Step.
+
+  This Step, however, does NOT generate your build: to create a `.app` file, you need the right step. For example, `Xcode build for simulator` Step or any other Step can generate a build. Add task URL
+website: https://github.com/autifyhq/bitrise-steps-autify-upload
+source_code_url: https://github.com/autifyhq/bitrise-steps-autify-upload
+support_url: https://github.com/autifyhq/bitrise-steps-autify-upload/issues
```
[the remark this answers is not in the document]
Comment (sotayamashita) [reply steps/autify-upload/0.0.1/step.yml:11]: ref: fixed: https://github.com/autifyhq/bitrise-step-autify-upload/pull/12/files

File: steps/autify-upload/0.0.1/step.yml (hunk 2 above)
Comment (sotayamashita) [reply steps/autify-upload/0.0.1/step.yml:30]: ref:
- fixed at original repo. https://github.com/autifyhq/bitrise-step-autify-upload/pull/13

**Comment by tarappo** [comment]
@lpusok 
Thanks for the review.
We fixed it, so could you please review this PR again.

File: steps/autify-upload/0.0.1/step.yml (hunk 3 above)
Comment (tarappo) [reply steps/autify-upload/0.0.1/step.yml:17]: Thanks.
We will remove it next release.

**Review by lpusok — APPROVED** [review]

**Pull request merged**